auxiliary/auxiliary.go

```diff
@@ -13,6 +14,85 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+// ParseArgs parses the command line arguments and returns the
+// user's configuration options for the caller to then use.
+func ParseArgs(args []string) (string, error) {
+	helpMessage := fmt.Sprintf(`Usage: %s [arguments]
```

Comment:
if an invalid argument is supplied, this help message will insinuate it's valid. Make it static.